_config.yml

```diff
@@ -288,3 +288,6 @@ defaults:
       comments: # true
       share: false
       related: false
+
+theme: jekyll-theme-midnight
+baseurl: /cf-networking-onboarding-1
```

Comment:
Are you able to run the site with this line? It breaks the site on my machine. If we wanted to update the base URL we would need to move all of the files and update lots of links I think. 

Can you remove it from this PR and if you want to introduce it, do it in a different PR with fixes for the other things mentioned above?

Reply:
Done ...
Don't know where the theme came from, but the base url is required (my fork is named different for some reason).